packages/app/src/App.tsx

```diff
@@ -93,9 +98,60 @@ import { ApacheAirflowPage } from '@backstage/plugin-apache-airflow';
 import { PermissionedRoute } from '@backstage/plugin-permission-react';
 import { catalogEntityCreatePermission } from '@backstage/plugin-catalog-common';
 
+const highlightOverride = {
+  BackstageHighlightedSearchResultText: {
```

Comment:
This is sweet!  TIL.

How difficult would it be to illustrate this in a storybook story?